tests/__tests__/auto-cleanup.js

```diff
@@ -0,0 +1,26 @@
+import { render } from '@testing-library/vue'
+import '@testing-library/jest-dom/extend-expect'
```

Comment:
same here (even though here we could use `.toHaveTextContent()` instead of manually checking `textContent`)